components/host-dashboard/reports-section/helpers.js

```diff
@@ -4,3 +4,7 @@ export const getActiveYearsOptions = host => {
   const activeYears = [...Array(currentYear - firstYear + 1).keys()].map(year => year + firstYear);
   return activeYears.map(year => ({ value: year, label: year })).reverse();
 };
+
+export const formatChartYAxisLabels = value => {
```

Comment:
Not all charts will have an amount for `y` axis. We can probably find a more explicit name. Suggestion:

```suggestion
export const formatAmountForLegend = value => {
```